Formula/warp.rb

```diff
@@ -0,0 +1,24 @@
+class Warp < Formula
+  desc "Secure terminal sharing with one simple command"
+  homepage "https://github.com/spolu/warp"
+  url "https://github.com/spolu/warp.git",
+    :tag => "v0.0.3",
+    :revision => "8a04cd7c95973e168a9bf2d2db6522574302164d"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["GOBIN"] = bin
+    ENV["GOPATH"] = buildpath
+    ENV["GOHOME"] = buildpath
+
+    mkdir_p buildpath/"src/github.com/spolu/"
+    ln_sf buildpath, buildpath/"src/github.com/spolu/warp"
+
+    system "go", "build", "-o", "#{bin}/warp", "github.com/spolu/warp/client/cmd/warp"
+  end
+
+  test do
+    system bin/"warp", "help"
```

Comment:
So ~anything else would rely on our servers being up and responding. Which is fine, if you think that's better?